rpm/resalloc.spec.tpl

```diff
@@ -292,8 +321,10 @@ ln -s "%{default_sitelib}/%{name}server" %buildroot%_homedir/project
 
 
 %if %{with python3}
+%if %{with sysusers_compat}
```

Comment:
Every time I am confused by with/without in RPM specfiles. But we want to do this only on old distributions, right?

Reply:
I disabled RHEL 7 CI.